dodoc/config.ts

```diff
@@ -0,0 +1,280 @@
+    if (
+      method.startsWith("constructor") ||
+      method.startsWith("fallback") ||
+      method.startsWith("receive")
+    )
+      specialMethods[method] = methods[method];
+    else normalMethods[method] = methods[method];
+  }
+
+  return [specialMethods, normalMethods];
+};
+
+const formatLinks = (textToFormat: string) => {
+  let formatedText: string = textToFormat;
+  [...textToFormat.matchAll(/\s\w+\s+http\S+/g)].forEach((element) => {
+    const tuple = element[0].trim();
+    const firstSpace = tuple.indexOf(" ");
+    const title = tuple.substring(0, firstSpace);
+    const link = tuple.substring(firstSpace).trim();
+    formatedText = formatedText.replace(tuple, `[**${title}**](${link})`);
+  });
+
+  return formatedText;
+};
+
+const formatTextWithLists = (textToFormat: string) => {
+  let formatedText: string = textToFormat;
+  [...textToFormat.matchAll(/\s-\s/g)].forEach((element) => {
+    formatedText = formatedText.replace(
+      element[0],
+      `\n\n${element[0].trim()} `
+    );
+  });
+  [...textToFormat.matchAll(/\s\d+\.\s/g)].forEach((element) => {
+    formatedText = formatedText.replace(
+      element[0],
+      `\n\n${element[0].trim()} `
+    );
+  });
+
+  return formatedText;
+};
+
+const replaceAll = (
+  textToFormat: string,
+  textToReplace: string,
+  replaceWith: string
+) => {
+  let formatedText = textToFormat;
+  while (formatedText.includes(textToReplace))
+    formatedText = formatedText.replace(textToReplace, replaceWith);
+  return formatedText;
+};
+
+const formatCode = (textToFormat: string) => {
+  let formatedText: string = textToFormat;
+  if (textToFormat.length > 75) {
+    if (textToFormat.split(",").length >= 2) {
+      const start = textToFormat.substring(0, textToFormat.indexOf("(") + 1);
+      const end = textToFormat.substring(textToFormat.indexOf(")"));
+      const middle = textToFormat.replace(start, "").replace(end, "");
+      formatedText = `${start}${middle
+        .split(",")
+        .map((elem) => `\n    ${elem.trim()}`)}\n${end}`;
+    } else {
+      const start = textToFormat.substring(0, textToFormat.indexOf(")") + 1);
+      const end = textToFormat.replace(start, "").trim();
+      formatedText = replaceAll(
+        `${start}${end.split(" ").map((elem) => {
+          if (elem.includes("(") || elem.includes(")")) return ` ${elem}`;
+          return `\n    ${elem}`;
+        })}`,
+        ",",
+        ""
+      );
+    }
+  }
+
+  return formatedText;
+};
+
+const formatBulletPointsWithTitle = (textToFormat: string, title: string) => {
+  if (textToFormat.length === 0) return "";
+  let formatedText: string = `**${title}**\n\n`;
+  if (textToFormat.startsWith("- ")) textToFormat = " " + textToFormat;
+  textToFormat.split(" - ").forEach((elem) => {
+    if (elem.trim().length !== 0) formatedText += `- ${elem.trim()}\n`;
+  });
+  return formatedText;
+};
+
+const genAdditionalInfo = (contract: string, code: string, type: string) => {
```

Comment:
Suggesting changing the function name

```suggestion
const generateAdditionalInfo = (contract: string, code: string, type: string) => {
```